Metallum/Classes/Model/Band.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Bands.swift
+//  Metallum
+//
+//  Created by AppleTree on 3/3/16.
+//  Copyright © 2016 OlhaF. All rights reserved.
+//
+
+import Foundation
+
+final class Band {
+    let name: String?
+    var shortDescription: String?
+    let formationYear: NSNumber?
+    var members: [Member]
+    var genres: [String]?
+    
+    init?(bandName: String?, shortInfo: String?, foundationYear: NSNumber?, membersOfBand: [Member], genres: [String]?) {
```

Comment:
this constructor is not helpful at all
create failable construction that takes JOSN object